.github/workflows/snapshoty.yml

```diff
@@ -12,6 +12,10 @@ on:
     - '*.asciidoc'
     - 'docs/**'
 
+permissions:
+  contents: read
+  actions: read
```

Comment:
what's the reason for `actions:read`? I don't recall we needed it

Reply:
I re-ran the script on this repo, and for some reason, this was added. 
I don't know why it was missing since the workflow doesn't have a top-level permissions block.

Need to dig into this.

Reply:
But yes, looking at the snapshoty-simple action, I also don't see a reason for it.

Reply:
```suggestion
```